hathor/stratum/stratum.py

```diff
@@ -735,10 +736,10 @@ class StratumFactory(ServerFactory):
     mined_txs: dict[bytes, Transaction]
     deferreds_tx: dict[bytes, Deferred]
 
-    def __init__(self, manager: 'HathorManager', reactor: Reactor = reactor):
+    def __init__(self, manager: 'HathorManager', reactor: Reactor | None = None):
         self.log = logger.new()
         self.manager = manager
-        self.reactor = reactor
+        self.reactor = reactor or get_global_reactor()
```

Comment:
Why make `reactor` an optional attribute?

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/888/commits/fc8b6716021533e56c7c806d56c7255e9a05d82a

Reply:
Thanks!